src/main/resources/jupyter/google_sign_in.js

```diff
@@ -0,0 +1,38 @@
+require.config({
+    "shim": {
+        "gapi": {
+            "exports": "gapi"
+        }
+    },
+    "paths": {
+        "gapi": "https://apis.google.com/js/platform"
+    }
+})
+
+/// TEMPLATED CODE
+var clientId = $(googleClientId);
+///
+
+require(['gapi'], function(gapi) {
```

Comment:
See https://docs.google.com/document/d/1hrxfYpZx5no1eWxzGyyiG4g-_lajaZb8wGqsfsF0qWo/edit?ts=5a03239f for some explanation of this code.